types/developerTestAccounts.ts

```diff
@@ -41,3 +41,17 @@ export type DeveloperTestAccountConfig = {
   salesLevel?: AccountLevel;
   contentLevel?: AccountLevel;
 };
+
+export type InstallOauthAppIntoDeveloperTestAccountResponse = {
+  authCodes: Array<{
+    developerQualifiedSymbol: {
+      projectName: string;
+      developerSymbol: string;
+    };
+    authCode: string;
+  }>;
+};
+
+export type TestPortalStatusResponse = {
```

Comment:
Saw your comment that these are the only response types, but what happens when this fails then? It seems like this is something that could fail pretty frequently since it wont work unless users backends are running

Reply:
Yeah confirmed with Caitlyn that these are the only two states that the BE can ever return. The only other possible state is a 500, but that wouldn't surface as a status type